app/helpers/risks_helper.rb

```diff
@@ -1,14 +1,6 @@
 # frozen_string_literal: true
 
 module RisksHelper
-  def heightened_observation_yes_no(level)
-    if Forms::Risk::RiskToSelf::OBSERVATION_LEVELS_REQUIRING_DETAILS.include?(level)
-      'yes'
-    else
-      'no'
-    end
-  end
-
   def full_observation_level_details(level, details)
```

Comment:
What is the point of full_observation_level_details method now? Can we remove this method as well?